### DISCUSSION TITLE
Do I need/there's a way to force float32?

### DISCUSSION THREAD

**Discussion by marcussacana** [comfyanonymous/ComfyUI#179] [Q&A] [answered]
Hi, I used other SD tools before, but due a bug in my GTX 1650 all SD tools asked me to force Float32, and I didn't find this option in the ComfyUI, Is there a way to force float32 or maybe isn't needed?

**Comment by comfyanonymous** [comment, marked as answer]
Assuming you use CheckpointLoaderSimple it load the model in fp32 mode if you have a 1600 series gpu.
